### ISSUE TITLE
Support Wyse60 terminal

### ISSUE THREAD

**Issue by fpietrosanti** [liftoff/GateOne#197]
I'm planning to use GateOne on a legacy application (based on OpenEdge installed on Linux) that use Wyse60-enabled terminal with 132x43 resolution .

I am wondering if GateOne is "agnostic" respect to the terminal emulation used or if it does need specific adaptation for "esoteric" terminal emulation mode like wyse 60.

Do you see any specific integration scenario with wy60 https://code.google.com/p/wy60/wiki/wy60 ?

Basically i would like to move my users, today using commercial windows terminal emulation softwares supporting wyse60, to using a Web Browser.

If you have any better idea if that's doable and/or it does require some specific adaptation, i could try to get some economic support to implement it in opensource.
